internal/netif/netif.go

```diff
@@ -75,7 +103,18 @@ func (m *netifManagerDefault) RemoveIPAddress() error {
 
 	l, err := m.LinkByName(m.devName)
 	if err != nil {
-		return xerrors.Errorf("could not get loopback interface %v", err)
+		var linkNotFoundErr netlink.LinkNotFoundError
+		if !errors.As(err, &linkNotFoundErr) && !m.manageInterface {
+			return xerrors.Errorf("could not get interface %s: %v", m.devName, err)
+		}
+		attrs := netlink.LinkAttrs{
+			Name: m.devName,
+		}
+		dummyLink := &netlink.Dummy{LinkAttrs: attrs}
+		err = m.LinkDel(dummyLink)
+		if err != nil {
+			return xerrors.Errorf("could add dummy interface %v", err)
```

Comment:
Copy & paste error in the error message. Please also add the device name and a colon.